src/status_im2/navigation/screens.cljs

```diff
@@ -66,6 +66,14 @@
                                            :to       0
                                            :duration constants/onboarding-modal-animation-duration}}}}})
 
+(def new-to-status-modal-animations
```

Comment:
perhaps we can put these in a separate file, e.g `transitions.cljs`. I believe we will have more of these in the future.

Reply:
Will do!

Reply:
Done